app/styles/colors.js

```diff
@@ -1,5 +1,5 @@
 export default {
-  blue: "#3366CC",
+  blue: "blue",
```

Comment:
No need for quotes

Reply:
Need quotes, it's just a javascript object